src/agent/CommandStatus.ts

```diff
@@ -79,8 +79,6 @@ class CommandStatus extends CommandPolykey {
               clientPort: response.clientPort,
               agentHost: response.agentHost,
               agentPort: response.agentPort,
-              publicKeyJWK: response.publicKeyJwk,
-              certChainPEM: response.certChainPEM,
```

Comment:
Does this fix any particular issue?

Reply:
i don't think so, just @addievo mentioned that it had little usage in status, so it was removed:
https://github.com/MatrixAI/Polykey-CLI/pull/45